nvm.sh

```diff
@@ -1120,12 +1156,16 @@ nvm() {
       shift
 
       nobinary=0
-      if [ "_$1" = "_-s" ]; then
-        nobinary=1
-        shift
-      fi
+      quiet=0
 
-      provided_version="$1"
+      while [[ "$#" -gt 0 ]]; do
+        case $1 in
+          -s) nobinary=1;;
+          -q) quiet=1;;
```

Comment:
any reason here not to do `quiet="--quiet"` or something more clear, so that the value in the install function would be more meaningful?